TTS/tts/layers/gst_layers.py

```diff
@@ -96,7 +96,7 @@ def __init__(self, num_heads, num_style_tokens,
         self.key_dim = embedding_dim // num_heads
         self.style_tokens = nn.Parameter(
             torch.FloatTensor(num_style_tokens, self.key_dim))
-        nn.init.orthogonal_(self.style_tokens)
+        nn.init.normal_(self.style_tokens, mean=0, std=0.5)
```

Comment:
Why do you use normal initialization? Orthogonal initialization should work better since we expect each style token to be different than the other.

Reply:
@erogol  I used normal initialization, because [Mellotron](https://github.com/NVIDIA/mellotron/blob/master/modules.py) uses it, as it was the repository with the most expressive results using GST, I found it interesting to keep what they implemented. If you still wanted, I can make the change!

Reply:
not really. If it is the case let it stay that way. Thx for the info :)

Reply:
Ok :)